Newsletters: add 230 (2022-12-14)

harding opened the pull request:
- [x] Update releases/RCs and add topic links @harding 
- [x] Client/Services section @bitschmidty 
- [x] Stack Exchange section @bitschmidty 
- [x] Bitcoin Core 25934 @Xekyo 
- [x] LDK 1835 @adamjonas 

Last regular newsletter of the year!

glozow commented on _posts/en/newsletters/2022-12-14-newsletter.md:
```diff
@@ -0,0 +1,242 @@
+  protocol optimized for creating [channel factories][topic channel
+  factories].  Channel factories allow multiple users to trustlessly
+  open multiple channels between pairs of users with only a single
+  transaction onchain.  For example 20 users could cooperate to create
```
```suggestion
  transaction onchain.  For example, 20 users could cooperate to create
```

glozow reviewed (commented):
Nice summaries!

fanquake commented on _posts/en/newsletters/2022-12-14-newsletter.md:
```diff
@@ -0,0 +1,242 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday (TODO: check for Bitcoin Core 24.0.1)
```
The binaries are now available, and the website should be updated to reflect this release today.

Xekyo commented on _posts/en/newsletters/2022-12-14-newsletter.md:
```diff
@@ -188,7 +188,40 @@ answers posted since our last update.*
+  Several answers explain that due to the lower cost of being able to generate many
+  Tor addresses as compared to IPv4 and IPv6 addresses, a Bitcoin node operator
+  exclusively using the Tor network could more easily be [eclipse attacked][topic eclipse
+  attacks] when compared to operating on only on clearnet or with a
```
```suggestion
  attacks] when compared to operating only on clearnet or with a
```

Xekyo reviewed (commented):
Very nice newsletter

jonatack commented on _posts/en/newsletters/2022-12-14-newsletter.md:
```diff
@@ -0,0 +1,287 @@
+    ![Illustration of two different jamming attacks](/img/posts/2020-12-ln-jamming-attacks.png)
+
+    With the current LN protocol, Alice is fundamentally limited to
+    concurrently forwarding a maximum of 483 pending HTLCs.  If she
```
Perhaps link `483 pending HTCLs` to https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#rationale-7

jonatack reviewed (commented):
A few comments in addition to the excellent feedback already posted.

harding commented:
@sipa 

> Is https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-December/021271.html worth mentioning (libsecp256k1 version 0.2.0 released)?

Absolutely!  Congrats on the release!  (For reference, it's our policy to announce all major releases, and most minor releases, for all the software listed in our regular *notable code and documentation changes* section, which includes libsecp256k1.)

harding commented:
Made all suggested edits (thanks @glozow @fanquake @Xekyo @sipa @jonatack !), reviewed other contributions (thanks @Xekyo @adamjonas and 2x @bitschmidty!), updated releases/RCs, and added topic links.  Thanks everyone for another great newsletter and another great year of regular newsletters.

If anyone reading this wants to help do an early review of the year-end newsletter, please send me an email or a message on IRC.

bitschmidty commented:
ACK on @harding's final edits, rebased and squashed. Thanks @harding @adamjonas @Xekyo @fanquake @glozow @jonatack for your contributions this week!
